content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: '.setUTCMinutes()'
+Desciption: 'brief, description of the setUTCMinutes function, examples'
+Subjects: 
+    - 'Computer Science' 
+    - 'Web Development'
+Tags: 
+    - 'Methods'
+    - 'Date'
+CatalogContent:
+    - 'introduction-to-javascript'
+    - 'paths/front-end-engineer-career-path'
+---
+
+The **.setUTCMinutes()** method of [Date](https://www.codecademy.com/resources/docs/javascript/dates) instances changes the minutes for this instance according to **universal time (UTC)**.
+
+## **Syntax(JS)**
+```js
+    setUTCMinutes(minutes)
+    setUTCMinutes(minutes, seconds)
+    setUTCMinutes(minutes, seconds, ms)
+```
+## Parameters
+`minutes`
+<br/>
+An integer value between 0 and 59 representing the minutes.
+<br/>
+
+`seconds`
+<br/>
+An integer value between 0 and 59 representing the seconds. If `seconds` is specified, you must also specify `minutes`.
+<br/>
+
+`ms`
+<br/>
+An integer value between 0 and 999 representing the milliseconds. If `ms` is specified, you must also specify `minutes` and `seconds`.
+<br/>
+<br/>
+<br/>
```

Comment:
```suggestion

- `minutes`: An integer value between 0 and 59 representing the minutes.
- `seconds`: An integer value between 0 and 59 representing the seconds. If `seconds` is specified, you must also specify `minutes`.
- `ms`: An integer value between 0 and 999 representing the milliseconds. If `ms` is specified, you must also specify `minutes` and `seconds`.

```

Reply:
We do not use HTML in the Markdown. Also this is presented better as a bulleted list.